test/low_energy_approx_chimera.jl

```diff
@@ -0,0 +1,132 @@
+using LinearAlgebra
```

Comment:
This does not look like a test file

Reply:
so where should I move it? outside the repo, or in the new folder,

Reply:
shall I make a performance_test folder